communication/mavsdkvehicleserver.h

```diff
@@ -47,6 +47,7 @@ public slots:
     void resetHeartbeat();
     void switchAutopilotID(const float autopilotID);
     void rxRtcmData(const QByteArray rtcmData);
+    void shutdownOrRebootOnboardComputer(int value);
```

Comment:
"int value" is not self-explanatory and allows too many unused options. As it essentially only are two options, it could, e.g., be a "bool isShutdown": true = shutdown, false = reboot.